tests/Query/SelectTest.php

```diff
@@ -254,6 +255,17 @@ public function testOrderByReset()
         $this->assertParams([], $select);
     }
 
+    public function testOrderByExpression()
+    {
+        $select = $this->factory
+            ->select()
+            ->from('users')
+            ->orderBy(express("FIELD(%s, 'off')", 'status'), 'DESC');
+
+        $this->assertSql("SELECT * FROM users ORDER BY FIELD(?, 'off') DESC", $select);
```

Comment:
Hmm, this isn't right. The SQL should have the field in the query, not a parameter.

Bad suggestion in my last review.

Changing `express(..., 'status')` to `express(..., identify('status'))` should do it.

Or, perhaps a more simple test that accomplishes the same thing:

```php
$select = $this->factory->select()->from('users')->orderBy(literal('RAND()'));
```